flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/rules/physical/batch/BatchPhysicalLegacySinkRule.scala

```diff
@@ -53,12 +52,14 @@ class BatchPhysicalLegacySinkRule extends ConverterRule(
             val dynamicPartIndices =
               dynamicPartFields.map(partitionSink.getTableSchema.getFieldNames.indexOf(_))
 
+            // TODO This option is hardcoded to remove the dependency of planner from
```

Comment:
Do we already have a follow-up ticket to introduce a separate option?

Reply:
@JingsongLi do you know if this option is still actively used?

Reply:
I know of others who use it because we don't have the `partition by` syntax on SQL.